assignments/closure.js

```diff
@@ -1,21 +1,49 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const number = 22;
 
-/* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
+function multiply48() {
```

Comment:
I hope you don't mind. I re-arranged the function a bit to make run it without errors.

const number = 22;
function multiply48() {
  return(number * 48); 
}; 

console.log(multiply48());

Reply:
A definition of closure that helped me: 

When you create a function, scope is created. Scope is where the JavaScript lives and is executed. The closure is a collection of all the variables in the scope that were created when the function was created.

When a function gets created and passed around or returned from another function, it carries a backpack with it. And in the backpack are all the variables that were in scope when the function was declared. Closure is the backpack that holds the variables for later.